src/components/ContextualMenu/ContextualMenu.less

```diff
@@ -8,19 +8,20 @@
 
 .ms-ContextualMenu {
   .ms-font-m;
-  .ms-u-normalize;
+  .resetMargins;
+  .resetPadding;
```

Comment:
Did you find another bug that required removing `ms-u-normalize` and adding `resetMargins` and `resetPadding`?